docs/eventing/README.md

```diff
@@ -340,6 +337,12 @@ example.
 - [Install the Eventing component](#installation)
 - [Run samples](./samples/)
 
+## Sending Events: Sources vs Manual
+
+If you need a Source not covered by the available Source implementations, there is a [tutorial on writing your own Source](./samples/writing-a-source/README.md).
```

Comment:
Link `[available Source implementations]` to https://knative.dev/docs/eventing/sources ?

Reply:
This is referring to https://knative.dev/docs/eventing/samples/writing-a-source/

Reply:
Oh I see, yes good point

Reply:
Have added that link too